Code/Legacy/CrySystem/System.cpp

```diff
@@ -392,7 +392,7 @@ void CSystem::ShutDown()
 
     // Audio System Shutdown!
     // Shut down audio as late as possible but before the streaming system and console get released!
-    Audio::Gem::AudioSystemGemRequestBus::Broadcast(&Audio::Gem::AudioSystemGemRequestBus::Events::Release);
+    Audio::Gem::SystemRequestBus::Broadcast(&Audio::Gem::SystemRequestBus::Events::Release);
```

Comment:
This Release can be done in the Deactivate of `AudioSystemGemSystemComponent`
This would remove the dependency on System.cpp